pkg/controller/actuator.go

```diff
@@ -104,6 +107,12 @@ func (a *actuator) Reconcile(ctx context.Context, ex *extensionsv1alpha1.Extensi
 		return err
 	}
 
+	if gardencorev1beta1helper.TaintsHave(cluster.Seed.Spec.Taints, gardencorev1beta1.SeedTaintDisableDNS) ||
+		cluster.Shoot.Spec.DNS == nil {
+		a.logger.Info("no shoot dns service installed", "shoot", ex.Namespace)
```

Comment:
maybe add the reason here so that readers of the logs know why :)

Reply:
ping?

Reply:
already done, see commit below

Reply:
I was talking about the log message, not the comment in the code (which is also good to add, of course) :) - can you add it in the log msg as well + squash the commits?